src/Task/PhpLint.php

```diff
@@ -53,6 +53,10 @@ public function run(ContextInterface $context): TaskResultInterface
             ->notPaths($config['ignore_patterns'])
             ->extensions($config['triggered_by']);
 
+        if ($files->isEmpty()) {
+            return TaskResult::createPassed($this, $context);
```

Comment:
Can you change this to:
```php
return TaskResult::createSkipped($this, $context);
```

This way, it is consequent with the other tasks.

Reply:
No problem! Fix is on the way!